### ISSUE TITLE
Enabling workspace/aerospike causes fatal error 

### ISSUE THREAD

**Issue by winfried-van-loon** [laradock/laradock#855]
Enabling the Aerospike arg for Workspace causes a fatal error for PHP. This is tested for the PHP7.1 version, which is now the default.

Running `php -v` within the Workspace bash returns:
```
PHP Fatal error:  Cannot redefine class constant Aerospike:: in Unknown on line 0
```

**Comment by winfried-van-loon** [comment]
May not be a Laradock issue, there is an open issue about this in the Aerospike repo:
https://github.com/aerospike/aerospike-client-php/issues/118

**Comment by bestlong** [comment]
PHP_VERSION=71
If PHP_FPM_INSTALL_AEROSPIKE_EXTENSION and PHP_FPM_INSTALL_OPCACHE then
build failed

**Comment by winfried-van-loon** [comment]
Should be fixed now by #906. However if you're still experiencing the issue, feel free to leave a comment.

**Comment by JonayMedina** [comment]
Hi, i having error running docker-compose build --no-cache 

> ERROR [laradock-aerospike internal] load metadata for docker.io/library/aerospike:latest

> [laradock-aerospike internal] load metadata for docker.io/library/aerospike:latest:
> ------
> failed to solve: rpc error: code = Unknown desc = failed to solve with frontend dockerfile.v0: failed to create LLB definition: docker.io/library/aerospike:latest: not found
